### DISCUSSION TITLE
INFINITY/NEG_INFINITY consts for vectors

### DISCUSSION THREAD

**Discussion by LPGhatguy** [bitshifter/glam-rs#315] [Ideas]
Inline with `Vec3::ZERO,` `Vec3::NAN`, etc, it would be helpful to also have positive and negative infinity as constants. These values come up a lot when refining bounds on objects, like for calculating bounding boxes.

**Comment by bitshifter** [comment]
Yep, I think that makes sense.

**Comment by bitshifter** [comment]
Merged in #403
